src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -56,18 +64,17 @@ public void testDoPost_ExistingUser() throws IOException, ServletException {
     Mockito.when(mockRequest.getParameter("username")).thenReturn("test username");
     Mockito.when(mockRequest.getParameter("password")).thenReturn("test password");
```

Comment:
When would we be asking for the password on the profile page? Does this test still pass without this line? If so, lets remove it.

Reply:
Yes it still works!